index.bs

```diff
@@ -1701,6 +1721,40 @@ a [=moment=] |sourceTime|, and a [=duration=] |eventReportWindow|:
     1. Set |lastEnd| to |lastEnd| + |deadline|.
 1. Return |windows|.
 
+To <dfn>parse report windows</dfn> given a [=map=] |map|, a [=moment=] |sourceTime|,
+and a [=list=] of [=event-level report windows=] |default|:
+
+1. [=Assert=]: |default| is not [=list/is empty|empty=].
+1. Let |defaultEndDuration| be the [=duration from=] |sourceTime| and the [=event-level report window/end=] of the last element of |default|.
```

Comment:
Do we need to ensure that `default` is sorted? It may make sense to introduce a "wrapper type" like `event-level report window list` that is defined as a list of event-level report windows sorted by X.

Reply:
Wrapper type SGTM, let me add one momentarily.